package.json

```diff
@@ -52,12 +52,17 @@
   "engines": {
     "pnpm": "9.3.0"
   },
+  "pnpm": {
+    "overrides": {
+      "@microsoft/api-extractor>typescript": "5.5.3"
```

Comment:
`@microsoft/api-extractor` uses typescript in its dependencies, this is a recommended way to use same version of typescript